contrib/experimental_dict_builders/benchmarkDictBuilder/README.md

```diff
@@ -14,46 +14,107 @@ make ARG="in=../../../lib/dictBuilder in=../../../lib/compress"
 
 ###Benchmarking Result:
 
-d=8
-f=23
-freq[i] = 0 when dmer added to best segment
+For every f value for fast, the first one is optimize and the second one has k=200
 
 github:
-| Algorithm         | Speed(sec)    | Compression Ratio  |
-| ----------------- | ------------- | ------------------ |
-| nodict            | 0.000007      |  2.999642          |
-| random            | 0.150258      |  8.786957          |
-| cover             | 60.388853     |  10.641263         |
-| legacy            | 0.965050      |  8.989482          |
-| fastCover(opt)    | 84.968131     |  10.614747         |
-| fastCover(k=200)  | 6.465490      |  9.484150          |
+NODICT       0.000023       2.999642
+RANDOM       0.149020       8.786957
+LEGACY       0.854277       8.989482
+FAST15       8.764078       10.609015
+FAST15       0.232610       9.135669
+FAST16       9.597777       10.474574
+FAST16       0.243698       9.346482
+FAST17       9.385449       10.611737
+FAST17       0.268376       9.605798
+FAST18       9.988885       10.626382
+FAST18       0.311769       9.130565
+FAST19       10.737259       10.411729
+FAST19       0.331885       9.271814
+FAST20       10.479782       10.388895
+FAST20       0.498416       9.194115
+FAST21       21.189883       10.376394
+FAST21       1.098532       9.244456
+FAST22       39.849935       10.432555
```

Comment:
For each value of `f`, does approximately the same value for `k` get chosen?

Reply:
For github and hg-changelog, the chosen k value is very similar. For hg-commands and hg-manifest, the k value varies a lot across different f values.